packages/client/src/blockchain/chain.ts

```diff
@@ -513,17 +513,17 @@ export class Chain {
   /**
    * Gets the latest block in the canonical chain
    */
-  async getCanonicalSafeBlock(): Promise<Block> {
+  async getCanonicalSafeBlock(): Promise<Block | undefined> {
```

Comment:
Just writing releases notes and stumbled upon this: so this is a (type) breaking change.

I first tended to just leave and hope not too many people use this and the method below yet but now when thinking about it twice I would prefer of going (a small) extra round here and introduce two new methods `getCanonicalSafeBlockSafe()` and `getCanonicalFinalizedBlockSafe()` and keep the behavior of the current methods as is, also considering that both methods are just used once in client and can be easily updated.

Reply:
Ah wait, I just stumbled upon this again, but this is into our client package (and not in our blockchain package). So I think we can leave this as-is? It is very unlikely someone uses our client as an actual package dependency?

Reply:
Ah, you are right. I only saw `blockchain/*`. 🤦‍♂️

Sorry for the non-issue.